javascript/javascript1/week1/1.js

```diff
@@ -0,0 +1,121 @@
+// Age-IFy excericse
+
+const yearOfBirth = 1987;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log(
+  "You will be" + " " + age + " " + "years old in" + " " + yearFuture + "."
+);
+
+// Goodboy-oldboy excercise
+
+const dogYearOfBirth = 2017;
+
+let dogYearFuture = 2027;
+
+const dogYear = dogYearFuture - dogYearOfBirth;
+
+let shouldShowResultInDogYears = dogYear;
+
+if (shouldShowResultInDogYears) {
+  console.log(
+    "your dog will be" +
+      " " +
+      dogYear * 7 +
+      " " +
+      "dog years old in" +
+      " " +
+      dogYearFuture
+  );
+} else {
+  console.log(
+    "Your dog will be" +
+      " " +
+      dogYear +
+      " " +
+      "human years old in" +
+      " " +
+      dogYearFuture
+  );
+}
+
+//Housey pricey
+
+let volumeInMeters = [8 * 10 * 10, 5 * 11 * 8];
+let gardenSizeInM2 = [100, 70];
+const houseCost = [2500000, 1000000];
+
+//for peter
+
+const estimatedCostPeter = function () {
+  return volumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300;
+};
+
+if (estimatedCostPeter < houseCost[0]) {
+  console.log(
+    "Peter is paying" + " " + houseCost[0] - estimatedCostPeter + " " + "estra"
+  );
+} else if (estimatedCostPeter > houseCost[0]) {
+  console.log("peter saving" + " " + estimatedCostPeter - houseCost[0]);
+} else if (estimatedCostPeter === houseCost[0]) {
+  console.log("Correct Price");
+}
+
+// for julia
+
+const estimatedCostJulia = function () {
+  return volumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300;
+};
+
+if (estimatedCostJulia < houseCost[1]) {
+  console.log(
+    "Julia is paying" + " " + houseCost[1] - estimatedCostJulia + " " + "extra"
+  );
+} else if (estimatedCostJulia > houseCost[1]) {
+  console.log("Julia saving" + " " + estimatedCostJulia - houseCost[1]);
+} else if (estimatedCostJulia === houseCost[1]) {
+  console.log("Correct Price");
+}
```

Comment:
Have you checked if this code is working? 
It doesn't btw. 
I will paste the corrected version of your homework here but you should be aware that you shouldn't submit the homework that doesn't work 🙂 
```
// for Peter
const estimatedCostPeter = volumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300;
if (estimatedCostPeter < houseCost[0]) {
  console.log(`Peter is paying ${houseCost[0] - estimatedCostPeter} extra`);
} else if (estimatedCostPeter > houseCost[0]) {
  console.log(`Peter saving ${estimatedCostPeter - houseCost[0]}`);
} else {
  console.log("Correct Price");
}

// for Julia
const estimatedCostJulia = volumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300;
if (estimatedCostJulia < houseCost[1]) {
  console.log(`Julia is paying ${houseCost[1] - estimatedCostJulia} extra`);
} else if (estimatedCostJulia > houseCost[1]) {
  console.log(`Julia saving ${estimatedCostJulia - houseCost[1]}`);
} else {
  console.log("Correct Price");
}
```